src/components/add_review/add_review.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Avatar from '../page_header/avatar';
+import Logo from '../logo/logo';
+import ReviewBreadcrimbs from './breadcrumbs';
+import PageBackground from '../page_background/page_background';
+import AddForm from './add_review_form';
+
+
+const AddReview = ({movie}) =>
+  <section className="movie-card movie-card--full">
+    <div className="movie-card__header">
+      <PageBackground {...movie}/>
+      <h1 className="visually-hidden">WTW</h1>
+
+      <header className="page-header">
+        <Logo isLogoLight={true}/>
```

Comment:
булинь пропсы можно передавать не указывая true& 
```suggestion
        <Logo isLogoLight/>
```